extractor/filesystem/language/javascript/packagelockjson/extractor.go

```diff
@@ -1,37 +1,236 @@
-// Copyright 2024 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-// Package packagelockjson extracts package-lock.json files.
+// Package packagelockjson extracts yarn.lock files.
```

Comment:
IIUC this is about package-lock.json and copybara takes care of adding the license which is required.

Reply:
It also adds licenses when converting between github PRs and internal (e.g. all the python extractors did not have license headers when I initially added them as PRs). (#157)

I'm not sure what would happen because I am replacing an existing file though, I'll add the license headers back in just in case.

Reply:
Done, added license back in.